_pages/user_registry.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: User registry
+permalink: /user_registry
+horizontal: false
+nav: false
+---
+
+The user registry API is used to create and manage an AgStack user account. The API is built using the Python framework, Flask, where user information is stored. Receive an access token (`access_token`), refresh token (`refresh_token`), and domain ID (`domain_id`). For authentication, the `flask-jwt-extended` package is used to decode the JSON Web Token (JWT).
+
+
+<!--what does it mean to be blacklisted and whitelisted?  I recommend that we say blocklist and allowlist. https://www.techtarget.com/searchsap/news/252488768/SAP-sets-alternatives-to-master-slave-blacklist-whitelist-->
+
+
+View the following tasks that you can complete using the user registry API:
+
+* Sign up
+
+  To sign up as an user, complete the _Sign up_ form by entering your name, email, password, and phone number (optional). Information that is entered is verified to avoid creating duplicate accounts. 
+
+* Log in to AgStack
+
+* Update user account settings
+
+* Search for domains
+
+* Search for authority token 
+
+* Log out of your user account
+
```

Comment:
@Aanish97 May you review this file. I am still working on edits

Reply:
@dockerymick, seems good so far. Let me know if need any specific details.

Reply:
@dockerymick, seems good so far. Let me know if needed any details.